velox/core/QueryConfig.h

```diff
@@ -623,6 +623,13 @@ class QueryConfig {
   static constexpr const char* kStreamingAggregationMinOutputBatchRows =
       "streaming_aggregation_min_output_batch_rows";
 
+  /// If true, the streaming aggregation accumulates at least two input batches
+  /// and produce all the groups created from the previous input batch except
+  /// the last group.
+  static constexpr const char*
+      kStreamingAggregationTrySplitOutputAtInputBoundary =
+          "streaming_aggregation_try_split_output_at_input_boundary";
```

Comment:
In general, it is error prone to introduce configs that produce incorrect results if set incorrectly. Configs should only affect performance, but not correctness.

CC: @amitkdutta @xiaoxmeng @oerling @zacw7